### PR TITLE
Makes self heating cup ramen saner

### LINKED ISSUE #37030: Cup Ramen issues
(WEB REPORT BY: riekland REMOTE: 172.93.101.77:7777)
# Revision
a210ab8b23bba0c155be511a5553cdcbe6f91e6b

# Byond Version
515.1643

# Description

The description for the ramen asks for 12ml water, even though we use units instead. Giving it 5u activates 9u of hot ramen,heats it, and leaves 2u calcium hydroxide and 21u dry ramen. Adding any more water does nothing.

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[tweak][qol][consistency][grammar]

## What this does
removes the calcium oxide stuff in cup ramen, makes it a container reaction while still keeping the old "hot water causes it" one
now requires just 10u of water and not 12"ml"
Closes #37030.

## Why it's good
easier to use, no calcium dioxide left behind

## How it was tested
vend cup ramen, use water on it

## Changelog
:cl:
 * tweak: Self heating cup ramen now only requires the container for the reaction, and does not use calcium oxide within, now only requiring 10u of water to heat and not 12.
 * spellcheck: The millilitres on cup ramen's description have been changed to reference units, for consistency.

### CODE REVIEW

**Comment by SonixApache** [comment]
fixes #16788 the proper way, thank you so much

**Comment by hacker-on-steroids** [comment]
>In 80 BC, the Roman general Sertorius deployed choking clouds of caustic lime powder to defeat the Characitani of Hispania, who had taken refuge in inaccessible caves.[23] A similar dust was used in China to quell an armed peasant revolt in 178 AD, when lime chariots equipped with bellows blew limestone powder into the crowds.[24]

wypipo don season they ramen
a bit sad to see it go

**Comment by SECBATON-GRIFFON** [comment]
> > In 80 BC, the Roman general Sertorius deployed choking clouds of caustic lime powder to defeat the Characitani of Hispania, who had taken refuge in inaccessible caves.[23] A similar dust was used in China to quell an armed peasant revolt in 178 AD, when lime chariots equipped with bellows blew limestone powder into the crowds.[24]
> 
> wypipo don season they ramen a bit sad to see it go

it can still be found in self heating donk pockets so it's not entirely dummied out yet

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 1]
```diff
@@ -614,24 +614,18 @@
...
 /obj/item/weapon/reagent_containers/food/drinks/dry_ramen/heating //vendor version
-	name = "\improper cup ramen"
-	desc = "Just add 12ml water, self heats!"
-	icon_state = "ramen"
-/obj/item/weapon/reagent_containers/food/drinks/dry_ramen/heating/New()
-	..()
-	reagents.add_reagent(CALCIUMOXIDE, 2)
-	src.pixel_x = rand(-10, 10) * PIXEL_MULTIPLIER
-	src.pixel_y = rand(-10, 10) * PIXEL_MULTIPLIER
+	desc = "Just add 10u water, self heats!"
```
Comment (west3436) [inline code/modules/reagents/reagent_containers/food/drinks.dm]:
```suggestion
	desc = "Self-heating: just add 10u water!"
```
nit lmao

**Review by west3436 — APPROVED** [review]

File: code/modules/reagents/reagent_containers/food/drinks.dm (hunk 1 above)
Comment (SECBATON-GRIFFON) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: did seem pretty ESL, fixed

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 2]
```diff
@@ -614,24 +614,18 @@
 	reagents.chem_temp = COOKTEMP_READY
 
 /obj/item/weapon/reagent_containers/food/drinks/dry_ramen
-	name = "\improper cup ramen"
+	name = "\proper cup ramen"
```
Comment (jwhitak) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: Cup ramen is improper. (the big brand name for cup instant ramen is Cup Noodles)

Proper: "User has thrown cup ramen."
Improper: "User has thrown the cup ramen."

**Review by jwhitak — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/food/drinks.dm (hunk 2 above)
Comment (SECBATON-GRIFFON) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: if you say so

**Review by jwhitak — APPROVED** [review]

**Pull request merged**